### PR TITLE
Update light.markdown with default transition 

### PR DESCRIPTION
**Pull request by donkawechico**
## Proposed change

Merge along with [PR #36747 -- Support default transition in light profiles](https://github.com/home-assistant/core/pull/36747), if accepted. Documentation for proposed addition of a "transition" parameter to the "turn-on defaults" mechanism.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/36747
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by frenck — CHANGES_REQUESTED** [review]
There are unrelated commits in this PR. Please git rebase your PR onto the latest `next` branch 👍

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by Jazzmyne06** [comment]
How do I merge?

**Comment by klaasnicolaas** [comment]
> How do I merge?

You don't have to worry about that, we will merge it when the time comes 😉

File: source/_integrations/light.markdown
Comment (donkawechico) [inline source/_integrations/light.markdown:15]: @frenck Not a fan of the [oxford comma](https://andrewpegoda.com/2014/06/24/the-oxford-comma-plus-every-comma-rule-you-need-to-know/), I take it! 😝

Comment (frenck) [reply source/_integrations/light.markdown:15]: It is not about me being a fan or not, it is about it not being according to the project documentation standards. <https://developers.home-assistant.io/docs/documenting/standards>

Which are merely in place to keep things standardized.

Comment (donkawechico) [reply source/_integrations/light.markdown:15]: Doh! My bad. I even read that page last month when I started contributing and promptly forgot that detail. Apologies for the ribbing :)

**Pull request merged**
